src/zenml/cli/server.py

```diff
@@ -55,60 +60,62 @@ def explain_server() -> None:
 
 
 @server.command("up", help="Start a daemon service running the ZenServer.")
+@click.option(
+    "--ip-address", type=IP_ADDRESS, default="127.0.0.1", show_default=True
+)
 @click.option("--port", type=int, default=8000, show_default=True)
 @click.option("--profile", type=str, default=None)
-def up_server(port: int, profile: Optional[str]) -> None:
+def up_server(ip_address: str, port: int, profile: Optional[str]) -> None:
     """Provisions resources for the ZenServer."""
     from zenml.services import ServiceRegistry
     from zenml.zen_server.zen_server import ZenServer, ZenServerConfig
 
+    service_config = ZenServerConfig(
+        root_runtime_path=GLOBAL_ZENML_SERVER_CONFIG_PATH,
+        singleton=True,
+        ip_address=str(ip_address),
```

Comment:
Typing in the signature says ip_address: str - how come you need to cast to str here?

Reply:
You're correct. Mypy doesn't seem to catch these, as they are auto-generated by click anyway, but this is an IPv4/v6 address.